modules/dynamic-plug-in-development.adoc

```diff
@@ -9,6 +9,12 @@
 You can run the plugin using a local development environment. The {product-title} web console runs in a container connected to the cluster you have logged into.
 
 .Prerequisites
+* . Visit the link:https://github.com/openshift/console-plugin-template[`console-plugin-template`] repository containing a template for creating plugins in a new tab.
```

Comment:
I think this is fine. IIRC we used to have specific instructions for the "Use this template" flow in the repo, but at least this gives users a way to find the repo. The instructions won't work unless you've at least cloned it.